co-log-benchmark/co-log-benchmark.cabal

```
@@ -0,0 +1,32 @@
+name:                co-log-benchmark
+version:             0.1.0.0
+synopsis:            benchmarks of the co-log library
+description:         Set of benchmarks that can be used to verify
+                     that colog library does not introduce unnesessarily
+                     overhead for your use-case. And check that colog
+                     library behaves on par with other frameworks.
+license:             MPL-2.0
+license-file:        LICENSE
+author:              Alexander Vershilov
+-- maintainer:          alexander.vershilov@gmail.com
+copyright:           2018 Kowainik
+category:            Logging
+build-type:          Simple
+extra-source-files:  CHANGELOG.md
+cabal-version:       >=1.10
+
+source-repository    head
+  type:              git
+  location:          https://github.com/kowainik/co-log.git
+
+executable co-log-features-bench
+  main-is:             Main.hs
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base >=4.11 && <4.12,
```

This is needed to fix CI for different Cabal versions

```suggestion
  build-depends:       base >=4.10 && <4.13,
```